Source/MTT/ConvertMain.cs

```diff
@@ -166,124 +166,127 @@ private void BreakDown()
                 {
                     var line = StripComments(_line);
 
-                    var modLine = new List<string>(ExplodeLine(line));
-
-                    // Check for correct structure
-                    if ((line.StrictContains("enum") && line.Contains("{")) || (line.StrictContains("class") && line.Contains("{")))
-                    {                        
-                        throw new ArgumentException(string.Format("For parsing, C# DTO's must use curly braces on the next line\nin {0}.cs\n\"{1}\"", file.Name, _line));
-                    }
-
-                    // Enum declaration
-                    if (line.StrictContains("enum"))
+                    if (!line.IsPreProcessorDirective())
```

Comment:
Instead of wrapping everything in an `if` can you just change it to a continue statement.  ie.

```
...

if(line.IsPreProcessorDirective()) {
   continue;
}

...
```

Reply:
Indeed better, modified the code.